src/content/docs/en/guides/migrate-to-astro/from-create-react-app.mdx

```diff
@@ -0,0 +1,379 @@
+
+- Astro is [component-based](/en/core-concepts/astro-components/), and your markup structure will be similar before and after your migration.
+
+- Astro has [official integrations for React, Preact, and Solid](/en/guides/integrations-guide/react/) so you can use your existing JSX components. Note that in Astro, these files **must** have a `.jsx` or `.tsx` extension.
+
+- Astro has support for [installing NPM packages](/en/guides/imports/#npm-packages), including React libraries. Many of your existing dependencies will work in Astro.
+
+## Key Differences between Create React App and Astro
+
+When you rebuild your Create React App site in Astro, you will notice some important differences:
+
+- [MPA vs SPA](/en/concepts/mpa-vs-spa/): Create React App is a single-page application that uses `index.js` as your project's root. Astro is a multi-page site, and `index.astro` is your home page.
+
+- [`.astro` components](/en/core-concepts/astro-components/) are not written as exported functions that return page templating. Instead, you'll split your code into a "code fence" for your JavaScript and a body exclusively for the HTML you generate.
+
+- [Content-first](/en/concepts/why-astro/): Astro was designed for content-focused sites that opt-in to interactivity only as they need them. An existing Create React App app might be built for high client-side interactivity and may require advanced Astro techniques to include items that are more challenging to replicate using `.astro` components, such as dashboards.
+
+## Add your Create React App to Astro
+
+Your existing app can be rendered directly inside a new Astro project, often with no changes to your app's code.
+
+### Create a new Astro project
+
+Use the `create astro` command for your package manager to launch Astro's CLI wizard and select a new "empty" Astro project.
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  npm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  pnpm create astro@latest
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  yarn create astro@latest
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+### Add integrations and dependencies
+Add the React integration using the `astro add` command for your package manager. If your app uses Tailwind or MDX, you can add multiple Astro integrations using the same command:
+
+```sh
+# Using NPM
+npx astro add react
+npx astro add react tailwind mdx
+# Using Yarn
+yarn astro add react
+yarn astro add react tailwind mdx
+# Using PNPM
+pnpm astro add react
+yarn astro add react tailwind mdx
+```
+
+If your CRA requires any dependencies (e.g. NPM packages), then install them individually using the command line or by adding them to your new Astro project's `package.json` manually and then running an install command. Note that many, but not all, React dependencies will work in Astro. 
```

Comment:
I mention what I did above (re: having CRA in title) because of something like this. I don't think we introduced the acronym yet but I don't know how you prefer introducing them / when